test/docker-e2e/networks/config.go

```diff
@@ -0,0 +1,23 @@
+package networks
+
+// Config holds the configuration for connecting to an existing live chain
+type Config struct {
+	Name    string
+	ChainID string
+	RPCs    []string
+	Seeds   string
+}
+
+// NewMochaConfig returns a Config for the mocha testnet
+func NewMochaConfig() *Config {
```

Comment:
with these configs, you can wire up with an existing network, so you can deploy full nodes alongside them to test syncing functionality.